tests/README.md

```diff
@@ -0,0 +1,18 @@
+Tests are currently available for two different software token implementations,
+NSS's softokn and softhsm (with some limitations).
+
+The easiest way to configure things to run manual tests is to simply execute next command from project's root directory:
+```bash
+meson test -C builddir
+```
+Be aware that previous command might change if a different build directory has been selected. Previous command will create two temporary directories with each token's configurations.
+
+After `meson test` command has run, set up the environment with:
```

Comment:
replace with:
`If you want to manually test some commands against one of the supported tokens, execute the test suite first and then source the necessary environment variables via:`